### PR TITLE
package.json and .gitignore

### PR DESCRIPTION
**Pull request by asconwe**
Checkittttt

### CODE REVIEW

**Review by John-Hong — COMMENTED** [review]
eww

**Review by narst20 — COMMENTED** [review]
Should this have heroku information?

**Comment by asconwe** [comment]
Heroku would go somewhere else...

**Comment by asconwe** [comment]
@John-Hong no way

File: server.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ var methodOverride = require('method-override');
 var bodyParser = require('body-parser');
 var exphbs = require("express-handlebars");
 //==============Local dependencies
-var db = require("./models");
-var controller = require('./controllers/controller') // Name controllers accordingly
+// var db = require("./models");
```
Comment (asconwe) [inline server.js:7]: Commented out references to db until we have sequelize stuff from @narst20

**Comment by asconwe** [comment]
@John-Hong @narst20 FYI - the controller.js file behaves like the Sequelize index.js file, so it requires all the js files in the controllers directory. So, in our server.js file, we just need to require controller.js and call it with controller(app), like on line 30 in server.js

Also - someone review my code, pleaaaase

**Review by narst20 — APPROVED** [review]

**Pull request merged**